src/AdminTab/AdminTab.tsx

```diff
@@ -42,7 +53,12 @@ export const AdminTab: FunctionComponent<AdminTabProps> = ({
         <div className="AdminTab-tables">
           <div className="AdminTab-tables-header">
             <Title level={3}>{t('Imported Tables')}</Title>
-            <Button onClick={onOpenImportDataTab} type="ghost" size="large" icon={<PlusOutlined />}>
+            <Button
+              onClick={onOpenImportDataTab}
+              type={highlightImport ? 'primary' : 'ghost'}
```

Comment:
It's fine to always have it as `primary`.

Reply:
On initial import the user will know what the button does. Having it change color suddenly is confusing.

Reply:
ok, let's keep it simple